Common/Core/TrackSelectionDefaults.h

```diff
@@ -26,7 +26,7 @@ TrackSelection getGlobalTrackSelection();
 
 // Default track selection requiring a particular Run 3 ITS matching
 TrackSelection getGlobalTrackSelectionRun3ITSMatch(int matching,
-                                                   TrackSelection::GlobalTrackRun3DCAxyCut passFlag = TrackSelection::GlobalTrackRun3DCAxyCut::Default);
+                                                   int passFlag = TrackSelection::GlobalTrackRun3DCAxyCut::Default);
```

Comment:
did you cross check that this does not give any subtle error? In principle if it compiles it should be fine, but have you tried to run it?

Reply:
Hi, yes I'm running with this since some weeks now